docs/decisions/008-themes.md

```diff
@@ -0,0 +1,54 @@
+# Themes
+
+Date: 2024-10-08
+
+Status: accepted
+
+## Context
+
+Since the very beginning, the Spirit Design System has been using the concept of inverted tokens and component variants
+to enable light content on dark backgrounds.
+
+However, this approach has several drawbacks:
+
+1. **Limited component support**: The inverted component variant need to be created manually for each component. This
+   leads to a limited set of components that support the inverted theme.
+2. **Number of tokens**: The inverted tokens are hard to maintain and can lead to inconsistencies. For example, the text
+   color token `text-primary-default` needs an inverted variant `text-primary-inverted-default`. This leads to a lot of
+   duplicated tokens and makes it hard to keep them in sync.
+3. **Manual composition**: In a larger inverted composition, all components need to be switched to the inverted variant
+   manually.
+4. **Dark mode (un)readiness**: The concept of the inverted tokens and component variants is not flexible enough to
+   support a dark mode. The name "inverted" is too specific and does not work for a dark mode where the inversion cannot
```

Comment:
maybe `shouldn't be applied`? Because it can be done from a technical point of view.